assets/js/components/notifications/ErrorNotifications.test.js

```diff
@@ -150,4 +154,70 @@ describe( 'ErrorNotifications', () => {
 		);
 		expect( container ).toMatchSnapshot();
 	} );
+
+	it( 'does render the redo setup CTA if initial Site Kit setup authentication is not granted', () => {
+		provideUserAuthentication( registry, {
+			authenticated: false,
+		} );
+		provideSiteInfo( registry, {
+			setupErrorRedoURL: '#',
+			setupErrorCode: 'access_denied',
+			setupErrorMessage:
+				'Setup was interrupted because you did not grant the necessary permissions',
+		} );
+
+		const { container } = render( <ErrorNotifications />, {
+			registry,
+		} );
+
+		expect( container ).toHaveTextContent( 'Setup was interrupted' );
+		expect( container ).toHaveTextContent( 'Redo the plugin setup' );
+	} );
+
+	it( 'does not render the redo setup CTA if it is not due to the interuption of plugin setup and no permission is temporarily persisted', () => {
```

Comment:
This one typo was missed, I'll fix it myself to save the round trip:

```suggestion
	it( 'does not render the redo setup CTA if it is not due to the interruption of plugin setup and no permission is temporarily persisted', () => {
```